Fakebook.Posts/Fakebook.Posts.DataAccess/Models/FakebookPostsContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace Fakebook.Posts.DataAccess.Models {
+    public partial class FakebookPostsContext : DbContext {
+        public FakebookPostsContext(DbContextOptions<FakebookPostsContext> options) : base(options) { }
+        public virtual DbSet<Post> Posts { get; set; }
+        public virtual DbSet<Comment> Comments { get; set; }
+        protected override void OnModelCreating(ModelBuilder modelBuilder) {
+            modelBuilder.Entity<Post>(entity => {
+                entity.ToTable("Post");
+                entity.Property(e => e.CreatedAt).HasDefaultValueSql("NOW()").ValueGeneratedOnAdd();
+                entity.Property(e => e.Picture).HasDefaultValue("").ValueGeneratedOnAddOrUpdate();
+            });
+            modelBuilder.Entity<Comment>(entity => {
+                entity.ToTable("Comment");
+                entity.Property(e => e.CreatedAt).HasDefaultValueSql("NOW()").ValueGeneratedOnAdd();
+                entity.HasOne(d => d.Post).WithMany(p => p.Comments).OnDelete(DeleteBehavior.Cascade);
```

Comment:
let's [explicitly configure relationships as required](https://docs.microsoft.com/en-us/ef/core/modeling/relationships?tabs=fluent-api%2Cfluent-api-simple-key%2Csimple-key#required-and-optional-relationships) when we want them required so the model is more self-documenting.

Reply:
Property is now explicitly required, and has a foreign key constraint.